src/BenchmarksApps/TechEmpower/PlatformBenchmarks/PlatformBenchmarks.csproj

```diff
@@ -17,24 +19,7 @@
 
   <ItemGroup>
     <PackageReference Include="Dapper" Version="2.0.123" />
-  </ItemGroup>
-
-  <ItemGroup Condition="$(TargetFramework) == 'netcoreapp3.1'">
```

Comment:
If we want to dump old TFMs, there's various #ifs we can remove (e.g. in RawDb)

Reply:
Oh good point, I was a bit lazy.
@sebastienros is there any reason to keep the old TFMs around?

Reply:
We can remove any customization lower than 6.0.

Reply:
So we still need to support running on net6.0 with the code in "main"?

Reply:
Since it's the current lts I prefer to keep it working, we might still need to create some kpis, or benchmark service releases.